### DISCUSSION TITLE
How to disable pop-up icon

### DISCUSSION THREAD

**Discussion by Plavozont** [sienori/simple-translate#363] [Q&A] [answered]
Because when it pops it overlaps the content on the page and disturbs the user)

**Comment by ilyapopovs** [comment, marked as answer]
Hi @Plavozont 

In settings there's "Don't display button or panel" option. Is it the one you are looking for?
<img width="692" alt="image" src="https://user-images.githubusercontent.com/16862411/143399370-5f611faa-553c-4707-b575-3d51e4faf3ca.png">

Extension settings can be accessed by pressing on the cog button.
<img width="141" alt="image" src="https://user-images.githubusercontent.com/16862411/143399732-b1cb0279-a490-416e-8331-299db1c0170b.png">

Also you might want to look into the "Translate on modifier key pressed" option - it will make the pop-up icon appear only when the specified key is pressed at the same time :)
